lemonades.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "twoxa (202)",
+			"oem": "OnePlus",
+			"device": "OnePlus 9R",
+			"filename": "crDroidAndroid-14.0-20231116-lemonades-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lemonades/10.x/crDroidAndroid-14.0-20231116-lemonades-v10.0-BETA.zip/download",
+			"timestamp": 1700165969,
+			"md5": "95f05e426d33a2cf69be37bf45deeda0",
+			"sha256": "de850de72024f48a0927991a5a2e6aa81dc6c1687d0e9b34025c1989a1b1779a",
+			"size": 1167761721,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/official-14-crdroid-10-x-for-oneplus-9r-lemonades.4636354/",
```

Comment:
update forum to reflect crDroid 10 changes

Reply:
Ohh i forget to update links😅
Done.